backend/src/main/java/io/debezium/configserver/service/ConnectorIntegratorBase.java

```diff
@@ -22,13 +25,35 @@
 import io.debezium.configserver.model.PropertiesValidationResult;
 import io.debezium.configserver.model.PropertyValidationResult;
 import io.debezium.connector.postgresql.PostgresConnectorConfig;
+import org.jboss.logging.Logger;
 
 public abstract class ConnectorIntegratorBase implements ConnectorIntegrator {
 
+    private static final Logger LOGGER = Logger.getLogger(ConnectorIntegratorBase.class);
+
     protected abstract ConnectorDescriptor getConnectorDescriptor();
 
     protected abstract SourceConnector getConnector();
 
+    public static List<String> enumArrayToList(EnumeratedValue[] input) {
+        List<String> result = new ArrayList<>();
+        for (EnumeratedValue value : input) {
+            result.add(value.getValue());
+        }
+        return result;
+    }
+
+    @Override
+    public Map<String, AdditionalPropertyMetadata> allPropertiesWithAdditionalMetadata() {
+        LOGGER.error(this.getClass().getName() + " must implement ConnectorIntegrator#getAllConnectorProperties!");
```

Comment:
Is that a left-over?

Reply:
No, it's the default implementation so I don't have to do it in the "temp" implementations and still have some warning it's not done yet. Would prefer an exception but then the `/connector-types` endpoint won't work as it would call this when listing all connectors and the exception would happen. Better ideas?

Reply:
Maybe just add a // TODO comment? I don't think there should be anything in the logs about this.

Reply:
Done